test/tests/functional/pbs_svr_dyn_res.py

```diff
@@ -502,148 +496,20 @@ def test_res_size_invalid_input(self):
-                                      suffix=".scr",
-                                      body=script_body)
-        self.du.chmod(path=fn, mode=0755, sudo=True)
-
-        resval = ['"' + resname[0] + ' ' + '!' + fn + '"']
-
-        self.setup_dyn_res(resname, restype, resval)
-
-        # Submit job
-        a = {'Resource_List.foo': 'abc'}
-        j = Job(TEST_USER, attrs=a)
-        jid = self.server.submit(j)
-
-        # The job shouldn't run
-        job_comment = "Can Never Run: Insufficient amount of server resource:"
-        job_comment += " foo (abc != 1.25)"
-        a = {'job_state': 'Q', 'comment': job_comment}
-        self.server.expect(JOB, a, id=jid, attrop=PTL_AND)
-
-    def test_res_string_array_invalid_input1(self):
-        """
-        Test invalid values returned from server_dyn_resource
-        script for resource type 'string_array'
-        Script returns a 'string' instead of type 'string_array'.
-        """
-        # Create a resource of type string_array
-        resname = ["foo"]
-        restype = ["string_array"]
-
-        # Prep for server_dyn_resource script
-        script_body = "echo red blue"
-
-        fn = self.du.create_temp_file(prefix="PtlPbs_str_arr",
-                                      suffix=".scr",
-                                      body=script_body)
-        self.du.chmod(path=fn, mode=0755, sudo=True)
-
-        resval = ['"' + resname[0] + ' ' + '!' + fn + '"']
-
-        self.setup_dyn_res(resname, restype, resval)
-
-        # Submit job
-        a = {'Resource_List.foo': '"red,blue"'}
+        a = {'Resource_List.foobar': '2gb'}
         j = Job(TEST_USER, attrs=a)
         jid = self.server.submit(j)
 
-        # The job shouldn't run
-        job_comment = "Can Never Run: Insufficient amount of server resource:"
-        job_comment += " foo (red,blue != red blue)"
-        a = {'job_state': 'Q', 'comment': job_comment}
-        self.server.expect(JOB, a, id=jid, attrop=PTL_AND)
-
-    def test_res_string_array_invalid_input2(self):
-        """
-        Test invalid values returned from server_dyn_resource
-        script for resource type 'string_array'.
-        Script returns 'int' instead of type 'string_array'.
-        """
-        # Create a resource of type string_array
-        resname = ["foo"]
-        restype = ["string_array"]
-
-        # Prep for server_dyn_resource script
-        script_body = "echo 1,2,3"
-
-        fn = self.du.create_temp_file(prefix="PtlPbs_str_arr",
-                                      suffix=".scr",
-                                      body=script_body)
-        self.du.chmod(path=fn, mode=0755, sudo=True)
-
-        resval = ['"' + resname[0] + ' ' + '!' + fn + '"']
-
-        self.setup_dyn_res(resname, restype, resval)
-
-        # Submit job
-        a = {'Resource_List.foo': '"1,2,3"'}
-        j = Job(TEST_USER, attrs=a)
-        jid = self.server.submit(j)
+        # Also check that "Script %s returned bad output"
```

Comment:
Change the %s to <scr> or <script> (and for other occurrences in the test suite)

Reply:
Fixed it.
[pbs_svr_dyn_res.log](https://github.com/PBSPro/pbspro/files/1926184/pbs_svr_dyn_res.log)